packages/vite/src/node/plugins/reporter.ts

```diff
@@ -276,3 +287,23 @@ function displaySize(bytes: number) {
     minimumFractionDigits: 2,
   })} kB`
 }
+
+function displayTime(time: number) {
+  // display: {X}ms
+  if (time < 1000) {
+    return `${time}ms`
+  }
+
+  time = time / 1000
+
+  // display: {X}s
+  if (time < 60) {
+    return `${time.toFixed(2)}s`
+  }
+
+  const mins = parseInt((time / 60).toString())
+  const seconds = time % 60
+
+  // display: {X}min {Y}s
+  return `${mins}min ${seconds < 1 ? '' : `${seconds.toFixed(0)}s`}`
```

Comment:
Should `min` be `m`? Otherwise it means `sec` for `s` too. It also seems like there would be a trailing space if `seconds` is `0`. I think this should fix it:

```suggestion
  // display: {X}m {Y}s
  return `${mins}m${seconds < 1 ? '' : ` ${seconds.toFixed(0)}s`}`
```